s3watcher/lambda/__tests__/EnvironmentConfig.test.ts

```diff
@@ -0,0 +1,48 @@
+import { readConfig } from "../lib/EnvironmentConfig"
+
+const OLD_ENV = process.env
+
+beforeEach(() => {
+  jest.resetModules() // most important - it clears the cache
+  process.env = { ...OLD_ENV } // make a copy
```

Comment:
This is a deep copy because otherwise pushing new values into the object during a test would modify the original copy stored in `OLD_ENV`. I've actually changed this to set the enviroment to be empty to isolate the tests from the user/build environment.